core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -248,29 +265,32 @@
 
             if (document.createElement('datalist') && window.HTMLDataListElement) {
                 const userField = form.querySelector('input[name="user"]');
+                let timer;
 
                 if (userField) {
                     userField.addEventListener('input', function () {
                         if (userField.value.length < 2) {
                             return;
                         }
-
-                        const body = {
-                            FORM_SUBMIT: 'datalist_members',
-                            REQUEST_TOKEN: form.querySelector('input[name="REQUEST_TOKEN"]').value,
-                            value: userField.value
-                        };
-
-                        const userList = form.querySelector('datalist[id="userlist"]');
-
-                        request('POST', form.action, new URLSearchParams(body), function () {
-                            userList.innerHTML = '';
-                            JSON.parse(this.response).forEach(item => {
-                                const option = document.createElement('option');
-                                option.value = item;
-                                userList.appendChild(option);
-                            });
-                        }, false);
+                        if (timer) {
+                            clearTimeout(timer);
+                        }
```

Comment:
```suggestion
                        clearTimeout(timer);
```
No need to check for `timer` as `clearTimeout()` allows its parameter to be undefined.

Reply:
True. But I'm not even sure this should be still in here. Since my commit #3790 has been reviewed, even if not merged yet. I guess I'd wait until it is merged and then modify my PR accordingly.